### DISCUSSION TITLE
Set up on windows

### DISCUSSION THREAD

**Discussion by MohamedSalahqa** [allure-framework/allure2#2672] [Questions & Support]
i downloaded the zip file extract it, and then wrote this command  ( allure -- version ) but noting responded that its not command

**Comment by baev** [comment]
Did you follow [the steps from documentation](https://allurereport.org/docs/install-for-windows/)?

**Reply by MohamedSalahqa** [reply]
Yes but I could not set scoop, So I will be thankful if you share those steps with me here
